integration/tests/create_channel.test.ts

```diff
@@ -71,8 +84,11 @@ describe('Create channel', () => {
             CONFIG.ADDRESSES.USDC_TOKEN_ADDRESS,
             depositAmount,
             {
-                initialAllocationAmounts: [depositAmount, BigInt(0)],
-                stateData: '0x',
+                initialState: convertRPCToClientState(
+                    createParsedResponseParams.state,
+                    createParsedResponseParams.serverSignature
+                ),
+                channel: convertRPCToClientChannel(createParsedResponseParams.channel),
```

Comment:
Maybe it will be more readable if `createChannel` and `depositAndCreateChannel` take `unsignedInitialState` and an explicit `serverSignature` as parameters?
Right now dev users have no way to understand that `initialState` must contain a signature from the server.
As a matter of fact, Clearnode returns a separate `server_signature` specifically to hint on this.